frontend/src/components/pages/ItemsPage.js

```diff
@@ -9,76 +9,121 @@ import Divider from '@material-ui/core/Divider';
+    };
 
     return (
-        <div className={classes.root}>
-            <Grid container spacing={3}>
-                <Grid item xs={5}>
-                    <img src={camera} width="100%" />
+        <div className={classes.root} id="item_page">
+            <Grid container spacing={3} className="image">
+                <Grid item xs={5} className="image">
+                    <img src={camera} width="75%" />
                 </Grid>
 
                 <Divider orientation="vertical" flexItem />
 
-                <Grid item xs={6}>
-                    <h1>NAME OF EQUIPMENT</h1>
-                    <h4>Equipment ID: d4n5c6</h4>
-                    <h4>Belongs to : Frame-X</h4>
-                    <h4>Total Quantity : 5</h4>
-                    <h3>AVAILABLE : YES</h3>
+                <Grid item xs={6} className="equipment">
+                    <div className="heading1">
+                        <h1>{data.nameOfEquipment}</h1>
+                    </div>
                     <h4>
-                        DESCRIPTION :Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
-                        eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim
-                        veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea
-                        commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit
-                        esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat
-                        cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id
-                        est laborum.
+                        Equipment ID : <p className={classes.heading4}>{data.equipmentId}</p>
                     </h4>
-                    <Grid container spacing={3}>
-                        <Grid item xs={4} className={classes.button}>
-                            <Button variant="contained" color="primary" href="#contained-buttons">
-                                MAKE REQUEST
-                            </Button>
-                        </Grid>
-                        <Grid item xs={4} className={classes.button}>
-                            <Button variant="contained" color="primary" href="#contained-buttons">
+
+                    <h4>
+                        Belongs to : <p className={classes.heading4}>{data.society}</p>
+                    </h4>
+                    <h4>
+                        Total Quantity : <p className={classes.heading4}>{data.quantity}</p>
+                    </h4>
+                    <h3>AVAILABLE : {available}</h3>
```

Comment:
Add conditional check for rendering availability after converting the available variable to boolean